src/modules/fstab/fstab.conf

```diff
@@ -12,6 +12,11 @@ mountOptions:
     default: defaults,noatime
     btrfs: defaults,noatime,space_cache,autodefrag
 
+# Mount options to use for the EFI System Partition. If not defined, it
+# takes either the value of *vfat*, or if also missing, *default* from
```

Comment:
This mentions *vfat*, but the code doesn't do anything like that. To match the code:

```
If not defined, it uses the settings for *default*, above.
```

Reply:
If you have an (existing) configuration without the new `efiMountOptions` like:
```
mountOptions:
    default: defaults
    btrfs: defaults,noatime,space_cache,autodefrag
    vfat: umask=0007
ssdExtraMountOptions:
    ext4: discard
    jfs: discard
    xfs: discard
    swap: discard
    btrfs: discard,compress=lzo
crypttabOptions: luks,keyscript=/bin/cat
```
It will then execute [line 305-306](https://github.com/calamares/calamares/pull/1206/files#diff-a321688334e9f490c6444e4d85aadb81R305) with the variable `filesystem=vfat`. 

`vfat: umask=0007` is just an illustrative example. I don't know, if any distributor has defined a `vfat` value.

The comment should reflect this behaviour (English is not my strength).

Another concern I have: I don't know what happens, if a distributor sets an empty value: `efiMountOptions: `. But I guess, that this is very unlikely.